src/app/mock-data.ts

```diff
@@ -1387,3 +1387,115 @@ export const fhir = {
   ]
 }
 };
+
+export const var1 = 
```

Comment:
I think these variables could be computed from the fhir.phq9 Questionnaire, replacing this part with just a few lines of code.  Also, since there are two Questionnaires in this mockdata file, naming the variables "var1", etc., makes it ambiguous as to which Questionnaire it is that should contain the variable.  So, maybe export an array, something like phq9Vars.

Reply:
Fixed